src/init.ts

```diff
@@ -0,0 +1,65 @@
+import fs from "fs";
+import mkdirp from "mkdirp";
+import { Lang } from "./lang";
+import { getBoundMessage } from "./messages";
+
+export interface InitCustomizeManifest {
```

Comment:
Thank you for your suggestion. I totally agree with you.
I fixed the code to use `CustomizeManifest` instead of `InitCustomizeManifest`.
Would you kindly check it?